test-frame-test/src/test/java/io/skodjob/testframe/test/integration/AbstractIT.java

```diff
@@ -20,6 +21,11 @@ public abstract class AbstractIT {
                 KubeUtils.labelNamespace(r.getMetadata().getName(), "test-label", "true");
             }
         });
+        KubeResourceManager.getInstance().addDeleteCallback(r -> {
+            if (r.getKind().equals("Namespace")) {
+                LoggerUtils.logResource("Deleted", r);
+            }
+        });
```

Comment:
Should there be something that can be checked in the test? For example incrementing some atomic integer and then checking, that it's really incremented or switching some flag. Because this will log `Deleted`, but it's not verified anywhere.

Reply:
Also I don't know how it's handled in the addCreateCallback case -> maybe it should have some check as well (for the labels).

Reply:
Well this is just example how to register callback. So I dont see any reason why to test this kind of example. And for label in create callback test is already done.

Reply:
But I will think about that later today🙂